Services/Dashboard/Block/classes/class.ilDashboardBlockGUI.php

```diff
@@ -440,8 +504,21 @@ public function executeCommand(): string
                 break;
 
             default:
-                if (method_exists($this, $cmd . 'Object')) {
-                    return $this->{$cmd . 'Object'}();
+                switch ($cmd) {
+                    case 'confirmedRemove':
+                        $form = $this->ui->factory()->input()->container()->form()->standard('', $this->getManageFields())->withRequest($this->http->request());
+                        $ids = [];
+                        foreach ($form->getData() as $group) {
```

Comment:
Please have a look if you can drop this `foreach` if you use `->withAdditionalTransformation` in line `486` instead. As this is part of input validation IMO this belongs to the managed fields.